src/FffGcodeWriter.cpp

```diff
@@ -309,6 +311,38 @@ void FffGcodeWriter::setConfigRetraction(SliceDataStorage& storage)
     }
 }
 
+void FffGcodeWriter::setConfigWipe(SliceDataStorage& storage)
+{
+    Scene& scene = Application::getInstance().current_slice->scene;
+    for (size_t extruder_index = 0; extruder_index < scene.extruders.size(); extruder_index++)
+    {
+        ExtruderTrain& train = scene.extruders[extruder_index];
+        WipeScriptConfig& wipe_config = storage.wipe_config_per_extruder[extruder_index];
+
+        wipe_config.retraction_enable = train.settings.get<bool>("wipe_retraction_enable");
+        wipe_config.retraction_config.distance = train.settings.get<double>("wipe_retraction_amount");
+        wipe_config.retraction_config.speed = train.settings.get<Velocity>("wipe_retraction_retract_speed");
+        wipe_config.retraction_config.primeSpeed = train.settings.get<Velocity>("wipe_retraction_prime_speed");
+        wipe_config.retraction_config.prime_volume = train.settings.get<double>("wipe_retraction_extra_prime_amount");
+        wipe_config.retraction_config.retraction_min_travel_distance = 0;
+        wipe_config.retraction_config.retraction_extrusion_window = 99999.9;
+        wipe_config.retraction_config.retraction_count_max = 9999999;
```

Comment:
You can use `<numeric_limits>` for these kind of 'largest value possible' numbers.

Reply:
Good point, thanks! This was fixed.